packages/react/src/components/Authenticator/SelectMfa/SelectMfa.tsx

```diff
@@ -0,0 +1,178 @@
+import React from 'react';
+import { authenticatorTextUtil, MfaType } from '@aws-amplify/ui';
+
+import { Flex } from '../../../primitives/Flex';
+import { Heading } from '../../../primitives/Heading';
+import { Radio } from '../../../primitives/Radio';
+import { RadioGroupField } from '../../../primitives/RadioGroupField';
+import { useAuthenticator } from '@aws-amplify/ui-react-core';
+import { useCustomComponents } from '../hooks/useCustomComponents';
+import { useFormHandlers } from '../hooks/useFormHandlers';
+// import { FormFields } from '../shared/FormFields';
+import { ConfirmSignInFooter } from '../shared/ConfirmSignInFooter';
+import { RemoteErrorMessage } from '../shared/RemoteErrorMessage';
+import { RouteContainer, RouteProps } from '../RouteContainer';
+import { toLower } from 'lodash';
+
+const { getChallengeText } = authenticatorTextUtil;
+
+function parseMfaTypes(types?: MfaType[]): string {
+  if(types) {
+    return types.join();
```

Comment:
If you're displaying this result to users (which may be redundant given they'll see them in the radio options), probably want to use `.join(', ')` instead.